packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/token-code.js

```diff
@@ -90,10 +90,7 @@ module.exports = class TokenCodeGroupingRule extends BaseGroupingRule {
       }
 
       if (this.bernoulliTrial(syncRolloutRate, subject.uniqueUserId)) {
-        return this.uniformChoice(
-          ['treatment-code', 'treatment-link'],
-          subject.uniqueUserId
-        );
+        return this.uniformChoice(['treatment-code'], subject.uniqueUserId);
```

Comment:
If the Sync rollout rate is 1 and there is only one possible group, you could get rid of lines 92-94 and just do a `return 'treatment-code';`